flink-core/src/main/java/org/apache/flink/configuration/CleanupOptions.java

```diff
@@ -128,19 +134,23 @@ public static String extractAlphaNumericCharacters(String paramName) {
     public static final ConfigOption<Integer> CLEANUP_STRATEGY_FIXED_DELAY_ATTEMPTS =
             ConfigOptions.key(createFixedDelayParameterPrefix("attempts"))
                     .intType()
-                    .defaultValue(1)
+                    .noDefaultValue()
```

Comment:
Why not set this to Integer.MAX_VALUE?

Reply:
I created FLINK-26350 to cover that issue

Reply:
Or you could just use `org.apache.flink.annotation.docs.Documentation.OverrideDefault` 🤷

Reply:
uh, thanks for the pointer. I didn't know about it. I will use that and close FLINK-26350 👍